src/Momento.Sdk/Responses/CacheListFetchResponse.cs

```diff
@@ -66,6 +71,8 @@ public Hit(_ListFetchResponse response)
             {
                 return new List<string>(values.Select(v => v.ToStringUtf8()));
             });
+
+            ListLength = checked((int)values.Count);
```

Comment:
what does `checked` do here? And what is the type of `values.Count`?

Reply:
I feel checked is not required here since we would not overflow the integral-limit in this case. But because the other Apis were using checked for ListLength, I was not sure if the check is applicable here or not. I can remove that. 

values.Count should return int by default. My bad, will correct that too.